src/content/docs/recipes/continuous-integration.mdx

```diff
@@ -86,13 +86,10 @@ lint:
         - biome ci --reporter=gitlab --colors=off > /tmp/code-quality.json 
         - cp /tmp/code-quality.json code-quality.json
```

Comment:
Question: when the `biome ci --reporter=gitlab --colors=off > /tmp/code-quality.json` line failes, it seems like gitlab-ci does not continue to the next line, so `cp /tmp/code-quality.json code-quality.json` is never run.

Why is the report first put into the `/tmp/`-dir only to be copied back into the working directory?

Why not directly this?

```bash
biome ci --reporter=gitlab --colors=off > code-quality.json
```

Reply:
It's necessary because the pipe creates the file, and biome can process JSON files. Biome will pick up the file, and emit parsing diagnostics for it.

<img width="704" height="285" alt="image" src="https://github.com/user-attachments/assets/b41ac213-cbd0-47c1-8cb0-938d373deb1d" />